package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@edx/studio-frontend",
-  "version": "0.5.0",
+  "version": "0.5.1",
```

Comment:
I went ahead and bumped the version in this PR. My intention is to make a release, but can certainly pull this into a separate PR there are other ones landing.